cfme/utils/appliance/implementations/ui.py

```diff
@@ -172,6 +172,9 @@ def _check():
             result = self.browser.execute_script(self.ENSURE_PAGE_SAFE, silent=True)
             # TODO: Logging
             return bool(result)
+        time.sleep(0.2)
+        # added this sleep as loading, and other elements that we are checking
+        # in ENSURE_PAGE_SAFE script appears on page after script is run
```

Comment:
**REQUIRED:** I don't see why this is needed, we have silent failures on here and wait for 2 seconds overall? Any failure will jsut cause a retry in 0.2 seconds.

Reply:
removed